.rubocop.yml

```diff
@@ -10,46 +8,48 @@ AllCops:
   TargetRubyVersion: 2.6
   NewCops: enable
   Exclude:
-    - 'spec/shared/lib/mrss/**/*'
+    - 'spec/shared/**/*'
```

Comment:
I am not sure whether it makes sense to enable the linter for specs. Let's see how it goes, though.

Reply:
I think we should, but yeah, we can see how it goes and discuss further. This highlighted change, though, just excludes the spec/shared folder explicitly because its a submodule, and should be linted elsewhere (if at all).